mk/db.mk

```
@@ -30,13 +30,13 @@ db-up: $(GO_OUTPUT)/dbmigrate  ## Start postgres database
 .PHONY: .db-health
 .db-health:
 	@echo -n "Checking database is ready: "
-	@$(DOCKER) container exists postgres
+	@$(DOCKER) container inspect postgres &> /dev/null
 	@$(DOCKER) exec postgres pg_isready
 
 .PHONY: .db-health-wait
 .db-health-wait:
-	@$(DOCKER) container exists postgres
-	@while [ "$$($(DOCKER) inspect -f '{{.State.Healthcheck.Status}}' postgres)" != "healthy" ]; do echo -n "."; sleep 1; done
+	@$(DOCKER) container inspect postgres &> /dev/null
+	@while [ "$$($(DOCKER) inspect -f '{{.State.Health.Status}}' postgres 2> /dev/null | $(DOCKER) inspect -f '{{.State.Healthcheck.Status}}' postgres 2> /dev/null)" != "healthy" ]; do echo -n "."; sleep 1; done
```

```suggestion
ifeq (podman,$(DOCKER))
DOCKER_HEALTH_PATH := .State.Healthcheck.Status
else
DOCKER_HEALTH_PATH := .State.Health.Status
endif
.PHONY: .db-health-wait
.db-health-wait:
	@$(DOCKER) container inspect postgres &> /dev/null
	@while [ "$$($(DOCKER) inspect -f '{{$(DOCKER_HEALTH_PATH)}}' postgres 2> /dev/null)" != "healthy" ]; do printf "."; sleep 1; done
```

Comments about the suggested change:
- Decide the `DOCKER_HEALTH_PATH` value depending on the `DOCKER` value.
- Use that in the command to check the healthy state.
- Update `echo -n "."` by `printf "."`. I got issues when running that command in macos (x86_64) `sh -c 'echo -n "."'` (ignoring `-n` option, and printing `-n .` with a new line at the end, instead of just `.` without new line at the end.